embedded_plugins/Admin-Controls.ts

```diff
@@ -271,6 +271,7 @@ function artifactOptions() {
   for (let i = MIN_ARTIFACT_TYPE; i < MIN_SPACESHIP_TYPE; i++) {
     options.push(html`<option value=${i}>${ArtifactTypeNames[i]}</option>`);
   }
+  options.push(html`<option value=${15}>AntiMatter Cube</option>`)
```

Comment:
This is hard coded bc Admin Controls uses a buggy version of the deployed packages and we don't have a good way to update them.